timers/context-timer-opentelemetry/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2016-2024 Talsma ICT
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>nl.talsmasoftware.context</groupId>
+        <artifactId>context-propagation</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <!-- Artifact identification -->
+    <groupId>nl.talsmasoftware.context.timers</groupId>
+    <artifactId>context-timer-opentelemetry</artifactId>
+    <name>Context timer (opentelemetry)</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <project.moduleName>${project.groupId}.metrics</project.moduleName>
```

Comment:
Wrong `<project.moduleName>` value? It should probably be `.opentelemetry` instead of `.metrics`.